File: app/src/main/java/ro/mdc_software/coinhivesdkandroidexample/activities/MainActivity.java
package ro.mdc_software.coinhivesdkandroidexample.activities;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

import ro.mdc_software.coinhivesdkandroidexample.R;

import butterknife.ButterKnife;
import butterknife.OnClick;

/**
 * @author Cosmin Mihu
 */
public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        ButterKnife.bind(this);
    }

    @OnClick(R.id.bActivityExample)
    public void onActivityExampleClicked() {
        startActivity(new Intent(this, CoinHiveActivityExample.class));
    }

    @OnClick(R.id.bFragmentExample)
    public void onFragmentExampleClicked() {
        startActivity(new Intent(this, CoinHiveFragmentExampleActivity.class));
    }
}

File: app/src/main/java/ro/mdc_software/coinhivesdkandroidexample/activities/MainActivity.kt
package ro.mdc_software.coinhivesdkandroidexample.activities

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*
import ro.mdc_software.coinhivesdkandroidexample.R

/**
 * @author Cosmin Mihu
 */
class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        buttonActivityExample.setOnClickListener {
            startActivity(Intent(this, CoinHiveActivityExample::class.java))
        }

        buttonFragmentExample.setOnClickListener {
            startActivity(Intent(this, CoinHiveFragmentExampleActivity::class.java))
        }
    }
}
